views/login.html

```diff
@@ -0,0 +1,128 @@
+  
+    <link rel="stylesheet" type="text/css" href="public/loginStyle.css">
+    <!-- OwlCarousel CDN for image slider-->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/OwlCarousel2/2.3.4/assets/owl.carousel.min.css"
+          integrity="sha256-UhQQ4fxEeABh4JrcmAJ1+16id/1dnlOEVCFOxDef9Lw=" crossorigin="anonymous" />
+    <link rel="stylesheet"
+          href="https://cdnjs.cloudflare.com/ajax/libs/OwlCarousel2/2.3.4/assets/owl.theme.default.min.css"
+          integrity="sha256-kksNxjDRxd/5+jGurZUJd1sdR2v+ClrCl3svESBaJqw=" crossorigin="anonymous" />
+    <!-- Font awesome CDN for icons-->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" integrity="sha512-iBBXm8fW90+nuLcSKlbmrPcLa0OT92xO1BIsZ+ywDWZCvqsWgccV3gFoRBv0z+8dLJgyAHIhR35VZc2oM/gI1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Century+Gothic&display=swap">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+</head>
+<body class="body">
+    <main class="login-block">
+        <div class="container">
+            <div class="row">
+              <div class="col-md-4 login-sec">
+                <div class="logo" >
+                    <img src="public/images/kainosDarkLogo.png" alt="kainosDarkLogo" style="width: 175px;">
+                </div>
+            <form class="login-form" action="/login" method="post">
+                  {% if errormessage %}
+                            <div class="alert alert-danger" role="alert">
+                                {{ errormessage }}
+                            </div>
+                  {% endif %}
+              <div class="form-group"> <!-- username input -->
+                <i class="fa fa-user" style="font-size: 12px"></i>
+                <label for="exampleInputEmail1" class="text-uppercase" style="font-family: sans-serif; font-size: 12px">Username</label>
+                <input type="text" id="username" name="username" class="form-control" placeholder="Enter your username" value="{{ username }}" required>
+              </div>
+              <div class="form-group"> <!-- password input -->
+                <i class="fa fa-lock" style="font-size: 12px"></i>
+                <label for="exampleInputPassword1" class="text-uppercase" style="font-family: sans-serif; font-size: 12px">Password</label>
+                <input type="password" id="password" name="password" class="form-control" placeholder="Enter your Password" value="{{ password }}" required>
```

Comment:
There is a risk here that you could leak passwords: 

`value="{{ password }}"`

Reply:
I dont think this leaks passwords, the value={{}} attribute does this not take the value the user inputs and puts it in the post form?